arch/x86/Kconfig

```diff
@@ -70,6 +70,14 @@ config X86_IAMCU
 	assembly code will need to be updated to conform to the new calling
 	convention.  If in doubt say N
 
+config MMU
```

Comment:
rename this to X86_MMU

CONFIG_MMU should be a generic define not specific to any architecture, we will get to it later once the arch-to-kernel code API takes shape.